ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/qos.cfg

```diff
@@ -112,6 +112,60 @@ qos profile uc_mc_queues_test
       !! Test guaranteed percent
       bandwidth guaranteed percent 10
 !
+qos profile wred_queues_test
+   !
+   tx-queue 1
+      !! Test no priority
+      bandwidth percent 50
+      no priority
+      random-detect drop minimum-threshold 1 kbytes maximum-threshold 10 kbytes drop-probability 100
+   !
+   tx-queue 2
+      bandwidth percent 10
+      priority strict
+      random-detect drop drop-precedence 2 minimum-threshold 2 kbytes maximum-threshold 200 kbytes
```

Comment:
```
ceos2(config-qos-profile-wred_queues_test)#   tx-queue 1
ceos2(config-qos-profile-wred_queues_test-txq-1)#      !! Test no priority
ceos2(config-qos-profile-wred_queues_test-txq-1)#      bandwidth percent 50
ceos2(config-qos-profile-wred_queues_test-txq-1)#      no priority
ceos2(config-qos-profile-wred_queues_test-txq-1)#      random-detect drop minimum-threshold 1 kbytes maximum-threshold 10 kbytes drop-probability 100
ceos2(config-qos-profile-wred_queues_test-txq-1)#   !
ceos2(config-qos-profile-wred_queues_test-txq-1)#   tx-queue 2
ceos2(config-qos-profile-wred_queues_test-txq-2)#      bandwidth percent 10
ceos2(config-qos-profile-wred_queues_test-txq-2)#      priority strict
ceos2(config-qos-profile-wred_queues_test-txq-2)#      random-detect drop drop-precedence 2 minimum-threshold 2 kbytes maximum-threshold 200 kbytes
% Incomplete command
```

Reply:
```
ceos2(config-qos-profile-wred_queues_test-mc-txq-2)# random-detect drop drop-precedence 2 minimum-threshold 2 kbytes maximum-threshold 200 ?
  kbytes  Set threshold in kbytes

ceos2(config-qos-profile-wred_queues_test-mc-txq-2)# random-detect drop drop-precedence 2 minimum-threshold 2 kbytes maximum-threshold 200 kbytes ?
  drop-probability  Set maximum drop probability rate

ceos2(config-qos-profile-wred_queues_test-mc-txq-2)# random-detect drop drop-precedence 2 minimum-threshold 2 kbytes maximum-threshold 200 kbytes drop-probability ?
  <1-100>  Specify maximum drop rate

ceos2(config-qos-profile-wred_queues_test-mc-txq-2)# random-detect drop drop-precedence 2 minimum-threshold 2 kbytes maximum-threshold 200 kbytes drop-probability 42 ?
  weight  Set WRED weight
  <cr>

```

Reply:
Agreed that this is an issue. Updated the schema